### PR TITLE
SLVS-1439 SLVS-1472 Show INFO and BLOCKER severity levels in rule descriptions

### PR DESCRIPTION
**Pull request by georgii-borovinskikh-sonarsource**
[SLVS-1439](https://sonarsource.atlassian.net/browse/SLVS-1439)

Part of MMF-4011

[SLVS-1439]: https://sonarsource.atlassian.net/browse/SLVS-1439?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/IssueViz/Helpers/IAnalysisSeverityToVsSeverityConverter.cs [hunk 1]
```diff
@@ -48,10 +48,14 @@ public __VSERRORCATEGORY ConvertFromCct(SoftwareQualitySeverity severity)
         {
             switch (severity)
             {
+                case SoftwareQualitySeverity.Blocker:
+                    return __VSERRORCATEGORY.EC_ERROR;
```
Comment (vnaskos-sonar) [inline src/IssueViz/Helpers/IAnalysisSeverityToVsSeverityConverter.cs:52]: This should be a warning as well. We don't want to block users at all.

**Comment by vnaskos-sonar** [comment]
This are the icons 😁 

2024-09-26 17_21_33-SLVS_Bound_VS2019 - Microsoft Visual Studio - Experimental Instance
2024-09-26 17_21_56-SLVS_Bound_VS2019 - Microsoft Visual Studio - Experimental Instance
2024-09-26 17_20_48-SLVS_Bound_VS2019 - Microsoft Visual Studio - Experimental Instance
2024-09-26 17_21_18-SLVS_Bound_VS2019 - Microsoft Visual Studio - Experimental Instance

File: src/IssueViz/Helpers/IAnalysisSeverityToVsSeverityConverter.cs (hunk 1 above)
Comment (gabriela-trutan-sonarsource) [reply src/IssueViz/Helpers/IAnalysisSeverityToVsSeverityConverter.cs]: Should it?

Comment (gabriela-trutan-sonarsource) [reply src/IssueViz/Helpers/IAnalysisSeverityToVsSeverityConverter.cs]: Blocker literally means "error". I think we should clarify this.

**Review by gabriela-trutan-sonarsource — APPROVED** [review]

**Pull request merged**